at-tanvo/Week3/Exercise3.rb

```diff
@@ -0,0 +1,42 @@
+class HtmlToText
+  def get(html)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/14415110195423c0a98d6a64a6adf49f44c4fce4/at-tanvo/Week3/Exercise3.rb#L2"></span>Metrics/PerceivedComplexity: Perceived complexity for get is too high. [10/8]</td>
    </tr>
  </tbody>
</table>